htlcswitch.go

```diff
@@ -204,11 +209,63 @@ out:
 		case pkt := <-h.htlcPlex:
 			numUpdates += 1
 			// TODO(roasbeef): properly account with cleared vs settled
-			switch pkt.msg.(type) {
+			switch msg := pkt.msg.(type) {
```

Comment:
The changes below are unnecessary from the PoV of the latest state within master.